CSA.md

```diff
@@ -97,19 +99,19 @@ Capitalized and highlighted words have the meanings given on the Cover Page or i
 
 10. ## Indemnification  
 
-    10.1 <span class="header_3">Protection by Provider.</span>  <span class="coverpage_link">Provider</span> will defend and hold harmless <span class="coverpage_link">Customer</span> from and against all <span class="coverpage_link">Provider</span> Covered Claims made by someone other than <span class="coverpage_link">Customer</span>, <span class="coverpage_link">Customer's</span> Affiliates, or Users, and indemnify <span class="coverpage_link">Customer</span> for all out-of-pocket damages, awards, settlements, costs, and expenses, including reasonable attorneys' fees and other legal expenses, that directly result from the <span class="coverpage_link">Provider</span> Covered Claim.
+    10.1 <span class="header_3">Protection by Provider.</span>  <span class="coverpage_link">Provider</span> will indemnify, defend and hold harmless <span class="coverpage_link">Customer</span> from and against all <span class="coverpage_link">Provider Covered Claims</span> made by someone other than <span class="coverpage_link">Customer</span>, <span class="coverpage_link">Customer's</span> Affiliates, or Users, and all out-of-pocket damages, awards, settlements, costs, and expenses, including reasonable attorneys' fees and other legal expenses, that arise from the <span class="coverpage_link">Provider Covered Claim</span>.
 
-    10.2 <span class="header_3">Protection by Customer</span>.  <span class="coverpage_link">Customer</span> will defend and hold harmless <span class="coverpage_link">Provider</span> from and against all <span class="coverpage_link">Customer</span> Covered Claims made by someone other than <span class="coverpage_link">Provider</span> or its Affiliates, and indemnify <span class="coverpage_link">Provider</span> for all out-of-pocket damages, awards, settlements, costs, and expenses, including reasonable attorneys' fees and other legal expenses, that directly result from the <span class="coverpage_link">Customer</span> Covered Claim.
+    10.2 <span class="header_3">Protection by Customer</span>.  <span class="coverpage_link">Customer</span> will indemnify, defend and hold harmless <span class="coverpage_link">Provider</span> from and against all <span class="coverpage_link">Customer Covered Claims</span> made by someone other than <span class="coverpage_link">Provider</span> or its Affiliates, and indemnify <span class="coverpage_link">Provider</span> for all out-of-pocket damages, awards, settlements, costs, and expenses, including reasonable attorneys' fees and other legal expenses, that directly result from the <span class="coverpage_link">Customer Covered Claim</span.
```

Comment:
Remove the second use of "indemnify"; add closing bracket to last span class for Customer Covered Claim; change "directly result" to "arise"

```suggestion
    10.2 <span class="header_3">Protection by Customer</span>.  <span class="coverpage_link">Customer</span> will indemnify, defend, and hold harmless <span class="coverpage_link">Provider</span> from and against all <span class="coverpage_link">Customer Covered Claims</span> made by someone other than <span class="coverpage_link">Provider</span> or its Affiliates, and all out-of-pocket damages, awards, settlements, costs, and expenses, including reasonable attorneys' fees and other legal expenses, that arise from the <span class="coverpage_link">Customer Covered Claim</span>.
```